wallet/src/lib.rs

```diff
@@ -7,6 +7,7 @@
 )]
 #![no_std]
 #![warn(missing_docs)]
+#![allow(clippy::uninlined_format_args)]
```

Comment:
I wouldn't mind leaving this lint enabled even if it's a bit pedantic. But no strong feelings about it either way so OK to allow for now.